explore/1608-forth/flib/stm32f1/i2c.fs

```
@@ -96,7 +96,7 @@ $40005800 constant I2C2
   10 bit I2C1-CR1 hbis!    \ ACK enable
 
   \ Wait for bus to initialize
-  i2c.timeout @ begin 1- dup 0= i2c-busy? 0= or until
+  i2c.timeout @ begin 1- dup 0= i2c-busy? 0= or until drop
```

it seems to me that this should either hang (or crash) if it can't initialize or it should return an error flag